src/client/components/MapHeader.tsx

```diff
@@ -307,6 +311,23 @@ const MapHeader = ({
           </Select>
         </Flex>
       </Box>
+      <Box sx={{ position: "relative", mr: "5px" }}>
+        <Button
+          sx={{ ...style.selectionButton }}
+          // eslint-disable-next-line
+          onClick={() => store.dispatch(toggleFind(!findMenuOpen))}
+        >
+          <Box
+            sx={{
+              borderBottom: findMenuOpen ? "solid 1px" : "none",
+              borderBottomColor: "muted",
```

Comment:
The white-on-white isn't showing up here - consider either switching to a dark border:
```suggestion
              borderBottomColor: "muted",
```

Reply:
Good catch.